### PR TITLE
SimAirIo.SimAirClient version 0.0.7-beta

### PR DESCRIPTION
**Pull request by chausner**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/29253)

### CODE REVIEW

File: manifests/s/SimAirIo/SimAirClient/0.0.7-beta/SimAirIo.SimAirClient.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SimAirIo.SimAirClient
+PackageVersion: 0.0.7-beta
```
Comment (ItzLevvie) [inline manifests/s/SimAirIo/SimAirClient/0.0.7-beta/SimAirIo.SimAirClient.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/s/SimAirIo/SimAirClient/0.0.7-beta/SimAirIo.SimAirClient.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SimAirIo.SimAirClient
+PackageVersion: 0.0.7-beta
```
Comment (ItzLevvie) [inline manifests/s/SimAirIo/SimAirClient/0.0.7-beta/SimAirIo.SimAirClient.installer.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/s/SimAirIo/SimAirClient/0.0.7-beta/SimAirIo.SimAirClient.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: SimAirIo.SimAirClient
+PackageVersion: 0.0.7-beta
```
Comment (ItzLevvie) [inline manifests/s/SimAirIo/SimAirClient/0.0.7-beta/SimAirIo.SimAirClient.yaml:5]:
```suggestion
PackageVersion: 1.0.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Make sure to update the version folder from `0.0.7-beta` to `1.0.0`.

File: manifests/s/SimAirIo/SimAirClient/0.0.7-beta/SimAirIo.SimAirClient.locale.en-US.yaml (hunk 1 above)
Comment (chausner) [reply manifests/s/SimAirIo/SimAirClient/0.0.7-beta/SimAirIo.SimAirClient.locale.en-US.yaml:5]: The download URL suggests it's not really version 1.0.0, though. I guess they do not set the version number in the installer correctly, i.e. the installer will always have the default of 1.0.0.

Comment (chausner) [reply manifests/s/SimAirIo/SimAirClient/0.0.7-beta/SimAirIo.SimAirClient.locale.en-US.yaml:5]: Should we still set this version number in the manifest?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
>Should we still set this version number in the manifest?

Yes, to prevent upgrade loops when doing `winget upgrade SimAirIo.SimAirClient`.

When 1.1 comes, we can add `DisplayVersion: 1.0.0`, and bring back `PackageVersion` to `0.0.7-beta`

**Review by ItzLevvie — APPROVED** [review]

**Comment by jamespik** [comment]
Why are we masking the .7-beta with 1.0? Was this an ARP issue?

**Comment by ItzLevvie** [comment]
> Why are we masking the .7-beta with 1.0? Was this an ARP issue?

Yes, it shows up as `1.0.0` rather than `0.0.7-beta`:
image
image
image

Since `0.0.7` is less than `1.0.0`, WinGet will always try to upgrade the package via the `winget upgrade SimAirIo.SimAirClient` even if you're already on `0.0.7-beta`.

**Pull request merged**